e2e-tests/tfm/cypress/fixtures/mock-gef-facilities.js

```diff
@@ -1,6 +1,23 @@
 const CONSTANTS = require('./constants');
 
+const facilityEndDateEnabled = Number(Cypress.env('GEF_DEAL_VERSION')) >= 1;
+
+/**
+ * Gets facility end date properties if enabled on default deal version
+ * @returns {Pick<import('@ukef/dtfs2-common').Facility,'isUsingFacilityEndDate' | 'facilityEndDate'> | {} } mock facility end date properties if enabled, or empty object if not
```

Comment:
As we've typed facility in `dtfs2-common`, we want to reference whatever type `isUsingFacilityEndDate` or `facilityEndDate` on `facility` in the type in common, as these should 1. be the same and 2. are linked

Reply:
I think this should just be the actual type, not the path.  In my opinion, if it's printed to the console, then we can see what the end result is, but this just looks confusing.  I think it should be isUsingFacilityEndDate | facilityEndDate | {}

Reply:
I agree with @Zainzzkk I think the "returns" documentation is overkill, especially considering it's not actually consuming anything from anywhere else, nor is it TypeScript, it's just conditionally creating an object with 2x facility fields (`isUsingFacilityEndDate`, `facilityEndDate`). I think it could just be the following:

```js
@returns {Facility}
```
Could even take a point of view - maybe it should have a vague `{object}` documentation since it's not returning a full facility and it could return an empty object

Reply:
I think it'd be best to remove the return type then in this case, as it could fall out of sync -- especially as it's for e2e tests anyway